salt/loader/__init__.py

```diff
@@ -11,6 +11,7 @@
 import re
 import time
 import types
+from collections import OrderedDict
```

Comment:
Does this get used anywhere... we removed `salt.utils.odict`, but didn't point any usage to the new `OrderedDict` from collections.

Reply:
Just looked at the whole file... I guess it wasn't being used anyway... or something.